File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-05/plagiarized/L6/04/Kasus5.kt
package IR

/**
 * @author C166AB8FBBBE55174E16A67530426A8A 19011F43894F06E2BA0CA74061427469
 */
object Kasus5 {
    
    fun reverse(number: Int) {
        if (number > 0) {
            print(number % 10)
            reverse(number / 10)
        }
    }
    
    @JvmStatic
    fun main(args: Array<String>) {
        print("Enter an integer: ")
        val input = Scanner(System.`in`)
        val number = input.nextInt()
        
        reverse(number)
        println("")
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/plagiarized/L5/04/Kasus5.ts
import { Scanner } from 'readline-sync';

function reverse(number: number): number {
    return number % 10;
}

function main(): void {
    console.log('Enter an integer: ');
    const input: Scanner = new Scanner();
    const number: number = input.nextInt();

    while (number > 0) {
        console.log(reverse(number));
        number /= 10;
    }
    console.log('');
}

main();

